include/seqan3/alignment/pairwise/detail/type_traits.hpp

```diff
@@ -83,6 +85,12 @@ template <typename configuration_t>
 struct alignment_configuration_traits
 {
 private:
+    //!\brief Transforms integral types to unsigned types and chooses the minimal viable uint for floating point types.
```

Comment:
float?

Reply:
yes, why not? The algorithm can also work with floating point types.

Reply:
Obviously, I would have not addded something like this if it wasn't needed. So there is a test already that performs this with floating point types. That broke. This fixes it. Note this is independent of SIMD stuff.